### PR TITLE
Pull excerpts for Penguin Random House titles 

### PR DESCRIPTION
**Pull request by leonardr**
This branch moves the Penguin Random House API client into the main body of the pipeline code. Previously that code was only used ahead of time, to generate rows for the "Master Metadata List" spreadsheet. Now it's also used during the pipeline, to retrieve excerpts of the selected PRH titles.

Currently nothing happens to the excerpts. A future branch will make sure an HTML excerpt gets the same treatment as an EPUB containing a single spine item.

There is no way to edit the excerpts, which means we will be posting web pages that are full of useless appendages like tables of contents. They're too big to fit into a spreadsheet cell, which was the original plan, and we don't have time to come up with an alternate solution (or, indeed, to do the edits).

### CODE REVIEW

**Comment by nodanaonlyzuul** [comment]
Hey @leonardr 

Can you take a moment to fix resolve this merge conflict?
👍  branch name. 
❤️ 🐧

File: bin/model.py [hunk 1]
```diff
@@ -221,6 +239,128 @@ def hash(self):
...
+
+        Since this is only run to make raw HTML look like it came
+        from an EPUB, we assume that there is only going to be one
+        content file, "content.html".
+        """
+        data = {"links": None,
+                "spine": [
+                    {"href": "content.html", "type": "text/html"},
+                ]
+                "resources": [],
```
Comment (nodanaonlyzuul) [inline bin/model.py]: Heyo - we need a "," between `spine`'s closing "]" and `"resource...`

File: bin/model.py [hunk 2]
```diff
@@ -221,6 +239,128 @@ def hash(self):
...
+# timestamp %s
+
+content.html
+
+NETWORK:
+*
+""" % datetime.datetime.utcnow().isoformat()
+
+    @property
+    def webapp_webmanifest
```
Comment (nodanaonlyzuul) [inline bin/model.py]: Heyo  - this should be `def webapp_webmanifest(self):` right?

**Comment by leonardr** [comment]
Sorry, that stuff was fixed in my follow-up branch but not in this one.

**Comment by leonardr** [comment]
This branch was not ready to go; it's been replaced by https://github.com/NYPL-Simplified/web-reader-landing-page/pull/176

**Pull request closed without merging**